scripts/custom_checks/docstr_examples.py

```diff
@@ -80,5 +80,5 @@ def fix_single_file(path: Path) -> str | None:
         return "\n".join(was_fixed)
 
 
-def format_docstring_examples(files: list[Path]) -> str:
+def format_docstring_examples(files: tuple[Path, ...]) -> str:
```

Comment:
Why is it a tuple? I'd think `Sequence[Path]` is more appropriate.

Reply:
➕